app.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'bubble'
+require "colorize"
+
+
+File.open("/Users/Chris/Code/class_projects/soapbox/soapbox/.soapbox", "w") do |file|
+  file << "@chris"
+end
+
+login_prompt = ""
+
+file = File.open("/Users/Chris/Code/class_projects/soapbox/soapbox/.soapbox")
+username_files = file.read
+puts "Continue as #{username_files} or press [n] to login as a different user."
+login_prompt = gets.chomp.downcase
+  if login_prompt == "n"
+    puts "What is your username?"
+    username = gets.chomp.downcase.to_s
+    user = Bubble.new(username: username)
+    user.save_user
+  else
+    username = "@chris"
+    user = Bubble.new(username: username)
+    user.save_user
+  end
+
+body = ""
+
+def entire_feed
+  d = Dir.glob("/Users/Chris/Dropbox/SoapBox/*")
+  d.map do |file|
+    r = File.open(file)
+    answers = {
+      body: File.basename(file),
+      username: r.read,
+      created_at: r.birthtime.strftime("%b %e, %l:%M %p")
+    }
+    nb = Bubble.new(answers)
```

Comment:
Here the assignment to `nb` is not needed. Since `Bubble.new` is that last thing that is happening and the `nb` variable is never used, it can just be left off.